magento/magento2
# Extension Attributes will add for Order Items

### Description (*)


Already for Order Extension attributes are adding, but for Order Item, it was not added. This fixes is for adding the extension attributes to Order Item Collection .

### Related Pull Requests


Fixes for https://github.com/magento/magento2/issues/14623

### Fixed Issues (if relevant)

1. Fixes magento/magento2#14623

### Manual testing scenarios (*)

1. Add plugin and extension_attributes.xml to see the result in Rest API collection.

### Questions or comments


### Contribution checklist (*)
